components/victron/text_sensor.py

```diff
@@ -16,6 +16,8 @@
 CONF_DEVICE_MODE = "device_mode"
 CONF_FIRMWARE_VERSION = "firmware_version"
 CONF_DEVICE_TYPE = "device_type"
+CONF_LOAD = "load_state"
```

Comment:
```suggestion
CONF_LOAD_STATE = "load_state"
```
Please name this consts 1:1 to their value.